### ISSUE TITLE
ExpandSubWorkflows parameter for metadata endpoint v30 regression 

### ISSUE THREAD

**Issue by samanehsan** [broadinstitute/cromwell#3073]
In Cromwell v30, the `expandSubWorkflows` parameter for the metadata endpoint is always false even when setting it to true through the swagger UI. If I run this curl command directly, I will get subworkflow metadata: `curl -X GET "https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1/d0b993ec-c4e2-4ffd-ad14-564f4b6b5b41/metadata?expandSubWorkflows=true" -H "accept: application/json"` 

<img width="1301" alt="screen shot 2017-12-15 at 5 53 24 pm" src="https://user-images.githubusercontent.com/6414394/34063664-76a20afa-e1c1-11e7-940d-9cac09407267.png">

**Comment by geoffjentry** [comment]
I betcha it is set to default to false. I've had bad luck w/ that in the past.

**Comment by katevoss** [comment]
@samanehsan have you tried it not through swagger?

**Comment by samanehsan** [comment]
@katevoss, yes if I use curl I get the subworkflow metadata but the issue is that it's not getting set to true through the swagger UI

**Comment by geoffjentry** [comment]
Updating to say that I ran across the other day and it was only a vague memory of this issue which led me to not go down a giant rabbit hole of WTF

**Comment by dinvlad** [comment]
Just ran into this as well!
